esterni/AnalisiDeiRequisiti/res/sections/requisiti.tex

```diff
@@ -26,139 +26,217 @@ \subsection{Requisiti funzionali}
+			\item posizione all'interno di una stanza
+		\end{itemize} & Capitolato - UC16	\\
+		\hline
+		R1F16.1&L'aggiunta di una postazione va salvata in Ethereum	&VE\_2021\_01\_01v.1.0.0 - UC16 	\\
+		\hline
+		R1F16.2&	Se si tenta di creare o modificare una postazione assegnandole un codice già in utilizzo, l'azione deve fallire e deve comparire un messaggio d'errore& VE\_2020\_12\_18v.1.0.0 - UC16.1 	\\
+		\hline
+		R1F16.3&Se si tenta di creare o modificare una postazione assegnandole un codice del tag già in utilizzo, l'azione deve fallire e deve comparire un messaggio d'errore	&VE\_2020\_12\_18v.1.0.0 - UC16.2 	\\
+		\hline
+		R1F16.4&Se si tenta di creare o modificare una postazione assegnandole una posizione già in utilizzo, l'azione deve fallire e deve comparire un messaggio d'errore	&VE\_2020\_12\_18v.1.0.0 - UC16.3 	\\
+		\hline
+		R1F17&L'amministratore deve poter eliminare una postazione	& Capitolato - UC17	\\
+		\hline
+		R1F17.1	&L'eliminazione di una postazione va salvata in Ethereum	& VE\_2021\_01\_01v.1.0.0 - UC17	\\
+		\hline
+		R1F18&L'amministratore deve poter modificare i dati e la posizione di una postazione	& Capitolato - UC18	\\
+		\hline
+		R1F18.1&	La modifica di una postazione va salvata in Ethereum& VE\_2021\_01\_01v.1.0.0 - UC18	\\
+		\hline
+		
+		R1F19&L'amministratore deve poter gestire le credenziali degli utenti	& Capitolato - UC19	\\
+		\hline
+		R1F19.1&L'amministratore deve poter visualizzare una lista delle credenziali di tutti gli utente del sistema	& Capitolato - UC19.1	\\
+		\hline
+		R1F19.2&L'amministratore deve poter creare nuove credenziali per accedere al sistema. Le informazioni contenute nel profilo di un utente sono: Nome, Cognome, Nome utente, Password e Email. Tra queste le informazioni necessarie per l'accesso sono password e nome utente	& Capitolato - VE\_2021\_01\_01v.1.0.0 - UC19.2	\\
+		\hline
+		R1F19.3&L'amministratore può modificare le credenziali degli utenti del sistema	&Capitolato - UC19.3 	\\
+		\hline
+		R1F19.4&L'amministratore deve poter eliminare credenziali degli utenti del sistema	& Capitolato - UC19.4	\\
+		\hline
+		R1F19.5&Il sistema deve poter memorizzare più credenziali per ogni categoria di utenti: amministratori, dipendenti e addetti	& Capitolato - VE\_2021\_01\_01v.1.0.0	\\
+		\hline
+		R1F20&L'amministratore deve poter esplorare la storia delle occupazioni delle postazioni	& Capitolato - UC20 - VE\_2021\_01\_04v.1.0.0	\\
+		\hline
+		R1F20.1&L'amministratore deve poter visualizzare in forma tabellare le occupazioni di postazioni da parte di un utente specifico, con date e orari di inizio e fine delle occupazioni e codici delle postazioni.	&Capitolato - UC20.1 - VE\_2021\_01\_04v.1.0.0	\\
+		\hline
+		R1F20.1.1&L'amministratore deve poter effettuare una ricerca tra le postazioni occupate da un utente, specificando il periodo all'interno del quale l'utente occupava la postazione. & 	Capitolato - UC20.1.1 - VE\_2021\_01\_04v.1.0.0\\
```

Comment:
L'amministratore deve poter effettuare una ricerca tra le postazioni occupate da un utente in un dato periodo